packages/saltcorn-data/base-plugin/types.js

```diff
@@ -506,7 +506,7 @@ const number_stepper = (name, v, attrs, cls, fieldname, id) =>
         type: "button",
         onClick: `$(this).prev().val(Math.min(${
           isdef(attrs.max) ? attrs.max : "Infinity"
-        },+$(this).prev().val()+1)).trigger('change');${attrs.onChange || ""}`,
+        },+$(this).prev().val()+$(attrs.stepsize))).trigger('change');${attrs.onChange || ""}`,
```

Comment:
},+$(this).prev().val()+${attrs.stepsize||1})).trigger('change');${attrs.onChange || ""}`,